client/components/application/BroaderReview.tsx

```diff
@@ -41,81 +41,90 @@ export const BroaderReview: React.FC<BroaderReviewProps> = ({ applicationId }) =
           enableReinitialize={true}
           key={applicationId}
         >
-          <Form>
-            <div className='open:bg-white border border-2 m-2 open:shadow-lg rounded-sm'>
-              <div className='leading-6 p-2 bg-gray-100 text-bcBluePrimary dark:text-white font-semibold select-none cursor-pointer'>
-                <div className='flex'>
-                  <div className='w-1/2 p-2'>Evaluation Board</div>
-                  <div className='w-1/2 flex justify-end'>
-                    {isLoggedInUser && (
-                      <Button variant='primary' customClass='py-2 ' type='submit'>
-                        Save
-                      </Button>
-                    )}
+          {({ isValid }) => (
+            <Form>
+              <div className='open:bg-white border border-2 m-2 open:shadow-lg rounded-sm'>
+                <div className='leading-6 p-2 bg-gray-100 text-bcBluePrimary dark:text-white font-semibold select-none cursor-pointer'>
+                  <div className='flex'>
+                    <div className='w-1/2 p-2'>Evaluation Board</div>
+                    <div className='w-1/2 flex justify-end'>
+                      {isLoggedInUser && (
+                        <Button
+                          variant='primary'
+                          customClass='py-2 '
+                          type='submit'
+                          disabled={!isValid}
```

Comment:
Working for me, looks great!
<img width="708" alt="image" src="https://user-images.githubusercontent.com/102998728/213240369-82df0fd8-d03f-4a36-a10d-a3dd645f1035.png">

Minor UX comment: the button still highlights  a bit even when disabled when someone hovers over it. Doesn't matter much in the end though 👍